science/digital_rf/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           cxx11 1.1
+PortGroup           python 1.0
+
+github.setup        MITHaystack digital_rf 2.6.0
+maintainers         @MITHaystack {mit.edu:rvolz @ryanvolz} {mit.edu:swoboj @jswoboda}
```

Comment:
Is @MITHaystack a human? It would seem not to be, in which case, that account should not be listed as a maintainer. Also, unless there's good reason not to (like a port where the update process is really delicate), the port should also list `openmaintaner` at the end to indicate that MacPorts project members are allowed to make minor tweaks without consulting the maintainers.

Reply:
@MITHaystack is monitored by one of the members of our IT team. They can help notify people if there is an issue.

Reply:
I'll take off @MIThaystack and put on openmaintainer then if that works with your system.